third_party/react-dates/build.sh

```diff
@@ -11,4 +11,4 @@ browserify \
 node ./third_party/react-dates/scope-require.js | \
```

Comment:
@rsimha It looks like there are four more dependencies that are only needed by this script. 

https://github.com/ampproject/amphtml/blob/e420fe7215fd73a1e4115b9fca9a768d7786c2d6/third_party/react-dates/scope-require.js#L18-L21

I tried making the change to include them here instead, but I can't run this script successfully. Were you able to run it yourself?

```
Error: Cannot find module 'babelify' from 'amphtml'
    at .nvm/versions/node/v14.16.0/lib/node_modules/browserify/node_modules/resolve/lib/async.js:137:35
```

Reply:
Good catch on those four deps. It has been a while since I ran those scripts. It's possible that the paths to `babelify` and friends still need to be hard-coded to the global install location?

**Here is the code as it is today:**
https://github.com/ampproject/amphtml/blob/aa3c1a04d106ae70615eb2ee70e2c1d34a633e5f/third_party/react-dates/build.sh#L1-L14

**And here is what it was when the paths were hard-coded:**
https://github.com/ampproject/amphtml/blob/97968f86397d2906c5000b4f0bd75674cb1bd39a/third_party/react-dates/build.sh#L1-L12

Reply:
That did not work, but I was able to run it by scoping a package: #33154